### PR TITLE
events: Add info on brainstorming session (June 22, 2023)

### PR DESCRIPTION
**Pull request by razvand**

### CODE REVIEW

File: website/docs/events/brainstorming-2023-06-22.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+* 15:00-15:10: Welcome
+* 15:10-16:00: Brainstorming: Are slides (still) useful in teaching and learning in the digital age?
+* 16:00-16:15: Break
+* 16:15-17:00: Breakout groups
+  1. Structuring a class digital content repository:
+     What should be the layout of digital contents for a class?
+     Where do slides, text, skeleton code, demos, images, go?
+  1. Public vs private content:
+     What content should be available to trainees, what should remain private (such as exam questions)?
```
Comment (StefanJum) [inline website/docs/events/brainstorming-2023-06-22.md:46]: Maybe detail a bit on the fact that we can have lets say part of the exam questions public, it can all be between nothing public and all public.
It feels like the question now is a bit strict, it would get answers in the form of `Private stuff: exam questions, homework solutions, public stuff: lecture items, lab items`, nothing in-between.

**Review by StefanJum — CHANGES_REQUESTED** [review]
Looks good, only one comment.

File: website/docs/events/brainstorming-2023-06-22.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# Brainstorming: Education in the Digital Age
+
+Good education is always a challenge, even more so in the digital age, where information is readily available and the potential of automation is disrupting existing methods.
+Questions come about:
+
+* What are good digital education materials?
```
Comment (teodutu) [inline website/docs/events/brainstorming-2023-06-22.md]:
```suggestion
* What makes good digital education materials?
```

File: website/docs/events/brainstorming-2023-06-22.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+* Which materials should be used live (during a class, lecture) or which should be used offline?
```
Comment (teodutu) [inline website/docs/events/brainstorming-2023-06-22.md]:
```suggestion
* Which materials should be used live (during a class, lecture) and which should be used offline?
```
This qurstion is a bit strange. It's as if you have a bunch of materials and you delegate some of them for live usage and the rest are left for offline. I'd reword it to "How to design materials for live usage and for individual study. Should they be separate?"

File: website/docs/events/brainstorming-2023-06-22.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+* What should happen face-to-face and what should happen online?
```
Comment (teodutu) [inline website/docs/events/brainstorming-2023-06-22.md:8]: 🍆, but leave it as it is.

File: website/docs/events/brainstorming-2023-06-22.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+* What are actions that are important to automate?
```
Comment (teodutu) [inline website/docs/events/brainstorming-2023-06-22.md]:
```suggestion
* Which actions are important to automate?
```

**Review by StefanJum — APPROVED** [review]
All good now.

File: website/docs/events/brainstorming-2023-06-22.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+* How do we structure content in repositories?
+* How is content available to trainees?
```
Comment (teodutu) [inline website/docs/events/brainstorming-2023-06-22.md]: I don't understand this one. Are you referring to how content is rendered or to which parts of it should trainees have accese (public vs private)?

I can deduce from the schedule that it's the latter, but this question alone is still confusing.

File: website/docs/events/brainstorming-2023-06-22.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+
+And many more.
+
+At Open Education Hub, we think about these items and we aim to create the future of education, given the challenges and opportunities of the digital age, and with the support of all those involved.
+
+On **Thursday, June 22, 2023, starting with 15:00 EEST**, we organize a brainstorming meeting to collect ideas and viewpoints from educators regarding education in the digital age.
```
Comment (teodutu) [inline website/docs/events/brainstorming-2023-06-22.md]:
```suggestion
On **Thursday, June 22, 2023, starting from 15:00 EEST**, we are organizing a brainstorming meeting to collect ideas and viewpoints from educators regarding education in the digital age.
```

File: website/docs/events/brainstorming-2023-06-22.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+     Where do slides, text, skeleton code, demos, images, go?
+  1. Public vs private content:
+     What content should be available to trainees, what should remain private (such as exam questions)?
+     Or what parts of types of content should be public and what parts should be private?
+     What content should be publicly available to everyone?
+  1. Automate everything:
+     What items are easiest / most important candidates for automation?
+     How easy is it for educators to use automation?
+  1. (Digital) Educational awards
+     What do we award during the educational process: good results, contributions, helping others?
```
Comment (teodutu) [inline website/docs/events/brainstorming-2023-06-22.md:53]: Specify that this is referring to students.

**Review by teodutu — CHANGES_REQUESTED** [review]
I made a bunch of suggestions, mainly rewordings and clarifications.

File: website/docs/events/brainstorming-2023-06-22.md (hunk 4 above)
Comment (razvand) [reply website/docs/events/brainstorming-2023-06-22.md:8]: love is love

File: website/docs/events/brainstorming-2023-06-22.md [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+At Open Education Hub, we think about these items and we aim to create the future of education, given the challenges and opportunities of the digital age, and with the support of all those involved.
+
+On **Thursday, June 22, 2023, starting from 15:00 EEST**, we are organizing a brainstorming meeting to collect ideas and viewpoints from educators regarding education in the digital age.
+Our goal is to extract best practices, guides and lessons learned in order to build up the methods, approaches and infrastructure for the future of education, a future that will be tied to digitalization and technology.
+
+In this 2.5 hours **in-person** discussion, we welcome anyone interested in sharing their ideas and in improving their (and others') digital educational contents, infrastructure and teaching methods.
+We will have drinks and sweets.
+
+We will spend about one hour discussing the usefulness of slides in the digital age.
+Then, for the next part, we will split in breakout groups to discuss several topics;
```
Comment (teodutu) [inline website/docs/events/brainstorming-2023-06-22.md]:
```suggestion
Then, for the next part, we will split into breakout groups to discuss several topics;
```

File: website/docs/events/brainstorming-2023-06-22.md [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+and, at the end, present those topics to the others.
+We will draw conclusions and then announce them to the greater education community.
+
+If you plan to attend, please fill [this form](https://forms.gle/kG3ioVqMJR8526sB9).
```
Comment (teodutu) [inline website/docs/events/brainstorming-2023-06-22.md:28]: Amend the agenda in the form to reflect the updated one below.

**Review by teodutu — CHANGES_REQUESTED** [review]
Just 2 small issues and then this will be good to go.

**Review by teodutu — APPROVED** [review]
